.travis.yml

```diff
@@ -6,7 +6,7 @@ perl:
 env:
   - "HARNESS_OPTIONS=j6"
 install:
-  - "cpanm -n Test::Pod Test::Pod::Coverage Data::Validate::Domain Data::Validate::IP YAML"
+  - "cpanm -n Test::Pod Test::Pod::Coverage Data::Validate::Domain Data::Validate::IP YAML::LibYAML"
```

Comment:
What does this line do in this context?

Reply:
See https://travis-ci.org/jhthorsen/mojolicious-plugin-openapi/jobs/558199927#L559

Reply:
So... nothing?

Reply:
Nothing to do with keeping up with the upstream changes in Mojo::Exception. With 8.22 this is no longer required.

Solely to get builds working
* [without it failed](https://travis-ci.org/jhthorsen/mojolicious-plugin-openapi/builds/558199926) a00d7e3
* [with it passed](https://travis-ci.org/jhthorsen/mojolicious-plugin-openapi/builds/558212799) c56b308